avro-fastserde/src/main/java/com/linkedin/avro/fastserde/backport/ResolvingGrammarGenerator.java

```diff
@@ -448,6 +451,16 @@ private static Symbol mkEnumAdjust(List<String> wsymbols,
     return Symbol.enumAdjustAction(rsymbols.size(), adjustments);
   }
 
+  private static Symbol mkEnumAdjustWithDefault(Schema writer, Schema reader) {
+    Resolver.EnumAdjust e = (Resolver.EnumAdjust) Resolver.EnumAdjust.resolve(writer, reader, GenericData.get());
+    Object[] adjs = new Object[e.adjustments.length];
+    for (int i = 0; i < adjs.length; i++) {
+      adjs[i] = (0 <= e.adjustments[i] ? new Integer(e.adjustments[i])
+          : "No match for " + e.writer.getEnumSymbols().get(i));
```

Comment:
this error msg could be a lot more useful: "reader schema {name} has no match for writer schema enum symbol XXX (value YYY)"

Reply:
From my understanding, this is not an error message.
Vanilla Avro sets the same enum symbol for unknown values
"
No match for ${enum value}
"